dsgrid/config/supplemental_dimension.py

```diff
@@ -19,3 +20,12 @@ class SupplementalDimensionModel(DimensionModel):
             title="mapping",
         ),
     ]
+
+
+class SupplementalDimensionsListModel(DSGBaseModel):
+    """Defines a list of supplemental dimensions."""
+
+    supplemental_dimensions: Annotated[
```

Comment:
Is this true, a container for both dimensions and mappings? If so, can we name it as such?

Reply:
This is directly analogous to this field: https://github.com/dsgrid/dsgrid/blob/main/dsgrid/config/project_config.py#L262. Would you prefer to call this `supplemental_dimensions_and_mappings` and lose the symmetry? Also, I could make an argument that a mapping is an attribute of a supplemental dimension and is therefore covered under this simpler name.

Reply:
If this is already in place in other config types, I am fine with keeping it as is. At one point, I think we have dimensions and mappings separate? So in my mental model, they are two distinct entities. Maybe it's time for me to update that.

Reply:
We store them separately in the registry. However, at some point we changed the project config definition to allow them to be specified in one object, as in this example: https://github.com/dsgrid/dsgrid-project-DECARB/blob/main/project/project.json5#L320.